cmd/proxy-server/server/server.go

```diff
@@ -219,7 +223,6 @@ func WithDebugPort(addr string) Option {
 				return
 			}
 		})
-		mux.Handle("/metrics", promhttp.Handler())
```

Comment:
I'd like to keep the ability to find raw metrics from the debug port. Could we either keep this registered here or change the index page (line 214) to link to the new port?

I mildly prefer keeping this line and making both the debug port and the metrics port serve /metrics.

Reply:
Agreed. I chose to keep this line so metrics can be accessed at either debug port or metrics port.